README.md

```diff
@@ -1,59 +1,49 @@
+![image](https://user-images.githubusercontent.com/24784631/216797451-f3d58b02-c900-4d37-9a3c-03ce48ffc411.png)
+
+## Table of Contents
 * [Introduction](#introduction)
 * [Requirements](#requirements)
 * [Install](#install)
 * [Data Files](#data-files)
+* [Common Issues](#common-issues)
 * [Reporting issues](#reporting-issues)
 * [Submitting fixes](#submitting-fixes)
-* [Copyright](#copyright)
-
-## Introduction
-
-LegionCore is an *MMORPG* Framework for Legion (26972) based on the uwow.biz core leaked early 2020.
-LegionCore itself has been derived from an old version of TrinityCore.
-
-TrinityCore is a *MMORPG* Framework based mostly in C++.
-
-TrinityCore is derived from *MaNGOS*, the *Massive Network Game Object Server*, and is
-based on the code of that project with extensive changes over time to optimize,
-improve and cleanup the codebase at the same time as improving the in-game
-mechanics and functionality.
-
-LegionCore development is completely open source; community involvement is highly encouraged.
-
-
-## Requirements
+* [Thank you](#thank-you)
 
-The software requirements for LegionCore are very similar to the one from TrinityCore.
-The main difference worth mentioning is that on Windows currently BOOST version 1.64 is required.
+# Introduction
+LegionCore is a **RPGMMO** framework for WOW Legion *(Build 26972)*. This core is based off of the UWOW core leak of 2020. Which was derived from an old version of [TrinityCore](https://github.com/TrinityCore/TrinityCore). LegionCore is completely opensource and is developed by the community. To submit a pull request please follow this template [here](submitting-fixes).
 
-Software requirements are available in the [wiki](https://www.trinitycore.info/display/tc/Requirements) for
-Windows, Linux and macOS.
+If you want you can join the community discord: [here](https://discord.gg/uaP2aeJ7sj).
 
+# Requirements
+ 
+ ### Important
+ The main difference to note is that LegionCore requires Boost (1.64.0), Visual Studio 2017 & 2017 build tools, and MySQL (5.6.51).
```

Comment:
I would expect any visual studio 2017+ to work, are you sure e.g. 2019 does not work?
From what I remember from installing on Windows, I never had to manually download the build tools, those come with VS2017?

Reply:
From my experience on Windows 10 build 22621.1105 I had to use VS2017 and install the build tools manually.

Reply:
It might've just been a one case thing?

Reply:
Also fine then, not too much to ask users anyway :)

Reply:
@LoveBeforT @T-R-I-X lets change it to: Visual Studio 2017 or later then.

Reply:
After that I am fine with merging it.

Reply:
You can merge now after reviewing one more time.